examples/flex-flows/eval-checklist/check_list.py

```diff
@@ -90,7 +82,10 @@ def __call__(self, answer: str, statements: dict):
         "correctness": "It contains a detailed explanation of ChatGPT.",
         "consise": "It is a consise statement.",
     }
-    flow = EvalFlow()
+
+    pf = PFClient()
+    connection = pf.connections.get("open_ai_connection", with_secrets=True)
```

Comment:
@eniac871 Looks like we still need the get connection with secret in this scenario.